src/layouts/BlogPostLayout.astro

```diff
@@ -42,6 +42,14 @@ import "../styles/global.css";
           <h1 class="text-3xl">{frontmatter.title}</h1>
         </header>
         <slot />
+        <style is:global>
+          h1 {
+            @apply pb-3.5;
+          }
+          h3, p {
+            @apply pb-2.5;
+          }
```

Comment:
If this is globally scoped will it not apply to all elements? 

Maybe we want this anyway, but I would keep it all in the global CSS file.

Reply:
[`63a03f7` (#28)](https://github.com/Lobstars-BVB/Lobstars/pull/28/commits/63a03f7fa0ca60e9bb26b5c13c4186dbd1395096)
wdyt? would be nice to be able to keep styles next to the compoenent, but no way to come aroudn the global thins afaicsee

Reply:
looks good!